drivers/i2s/Kconfig.stm32

```diff
@@ -25,3 +25,24 @@ config I2S_STM32_TX_BLOCK_COUNT
 	default 4
 
 endif # I2S_STM32
+
+menuconfig I2S_STM32_SAI
+	bool "STM32 MCU I2S controller driver"
+	default y
+	depends on DT_HAS_ST_STM32_SAI_ENABLED
+	depends on SOC_SERIES_STM32U5X
```

Comment:
Sorry if I missed this earlier, but this way to specify on U5 should be removed. DTS is used as the source of truth on that matter.
(Aim is to avoid that this becomes and endless (and useless) list of SOC_SERIES_FOO each time a new series support is added.)